web-ng/src/app/shared/models/project.model.ts

```diff
@@ -24,7 +24,7 @@ export class Project {
     readonly title: StringMap,
     readonly description: StringMap,
     readonly layers: List<Layer>
-  ) {}
+  ) { }
```

Comment:
Could you try using Prettier to reformat if you're not already? It should use eslint, which in turn will apply Google TypeScript formatting (gts).